### DISCUSSION TITLE
Error in importing prophet

### DISCUSSION THREAD

**Discussion by eeinnew** [teamalgoritma/community#114] [Q&A] [answered]
hello, I have some error in importing prophet
```
# load the library
from fbprophet import Prophet
# fit model
model_31 = Prophet()
model_31.fit(daily_total_qty)
```

output :
```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Input In [45], in <cell line: 2>()
      1 # load the library
----> 2 from fbprophet import Prophet
      3 # fit model
      4 model_31 = Prophet()

ModuleNotFoundError: No module named 'fbprophet'
```

p.s I have install the requirement txt  and `conda install -c conda-forge pystan=2.19.1.1`

**Comment by triarm** [comment, marked as answer]
Dear @eeinnew ,
Sepertinya package `fbprophet` belum terinstal pada environmentnya. Ibu dapat mencoba lakukan instalasi package tersebut dengan menjalankan command berikut di anaconda prompt: `conda install -c conda-forge fbprophet`
